.github/CONTRIBUTING.md

```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
 ### Articles
 
 As a bonus to this repository, there are associated articles for every script.
-However, the articles do take awhile to write, so you can help by writing them
+However, the articles do take a while to write, so you can help by writing them
 as well.
 
 Previously, we had hosted all of the articles on [The Renegade Coder][34], but
 we've since moved away from that to support open-source editing. Now, you
 can add and update any article you want.
 
-Currently, there are three types of articles you can write: code, project, and
+Currently, there are three types of articles you can write: code, project and
 language. You can find out more about this type of articles below.
 
 #### Code
 
-To get started, you'll want to create a new markdown file (i.e. `python.md`)
+To get started, you'll want to create a new markdown file (eg: `python.md`)
```

Comment:
This is interesting! I use i.e. a lot, so I had to look this one up: https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

Reply:
i.e., is used to clarify
Here, we are giving an example of file, ...
Hence, eg is more apt.